dist/index.html

```diff
@@ -22,51 +22,31 @@ <h1>Leaderboard</h1>
                 <div class="col-md-5">
                     <h4 class="d-flex justify-content-between align-items-center mb-3">
                         <span class="title">Recent scores</span>
-                        <span><button type="submit" class="btn btn-refresh">Refresh</button></span>
+                        <span><button type="button" class="btn btn-refresh">Refresh</button></span>
                     </h4>
                     <ul class="list-scores">
-                        <li class="list-score-item">
-                            <strong>Name: 100</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 20</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 50</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 78</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 125</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 77</strong>
-                        </li>
-                        <li class="list-score-item">
-                            <strong>Name: 42</strong>
-                        </li>
+
                     </ul>
                 </div>
                 <div class="col-md-4">
                     <h4 class="mb-3">
                         <span class="title">Add your score</span>
 
                     </h4>
-                    <form>
+                    <form id="form-submission">
                         <div class="input-group">
 
                             <div class="input-group">
 
                                 <span>
-                                <input type="text" placeholder="Your name">
+                                <input type="text" placeholder="Your name" id="add-name" required>
                             </span>
 
                                 <span>
-                                <input type="text" placeholder="Your score">
+                                <input type="text" placeholder="Your score" id="add-score" required>
                             </span>
 
-                                <button type="submit" class="btn btn-add">Submit</button>
+                                <button type="submit" class="btn btn-add" id="btn-submit">Submit</button>
```

Comment:
[Optional]
- [ ] It will be really good if you add some basic input validations for the score field. Currently, I can post anything as a score. Ideally, It should only accept numbers above zero

Reply:
I've also did this !
![demo-true](https://user-images.githubusercontent.com/6615394/132775379-af898743-bdd2-476f-bdc3-29c7630328b5.PNG)
![demo-error](https://user-images.githubusercontent.com/6615394/132775384-94a145c2-1f60-41d9-b3d0-8b61c97a11a1.PNG)
![demo-error2](https://user-images.githubusercontent.com/6615394/132775387-355cb63b-e85f-42b8-a184-1a64e379fe94.PNG)